src/main/java/lotto/domain/LottoPrice.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoPrice {
+
+    private final int price;
+    private final int ticketCount;
+
+    public LottoPrice(int price) {
+        this(price, price / 1000);
```

Comment:
```suggestion
        this(price, price / 1_000);
```
돈을 나타내는 숫자의 경우 천 단위를 구분하기 위해 "1_000"와 같이 언더바를 통해 구분할 수 있다.